content/functions/title.md

```diff
@@ -23,3 +23,12 @@ aliases: []
 ```
 {{title "BatMan"}}` → "Batman"
 ```
+
+Can be piped to combine with others.
+
+```
+{{ range $name, $items := .Site.Taxonomies.<YOUR_TAXONOMY> }}
+  <li><a href="{{ $.Site.BaseURL }}<YOUR_TAXONOMY>/{{ $name | urlize | lower }}">{{ $name  | humanize | title }} ({{ len $items }}</a>
```

Comment:
Building a url this way can be error prone. It can happen that users defines the base url as `www.example.com` without a trailing slash in the end. In this case `www.example.com<YOUR_TAXONOMY>....` isn't a valid url.

Instead use something like this:

    {{ printf "%s/%s" <YOUR_TAXONOMY> ($name | urlize | lower) | absURL }}

As you can see we build the path to the resource and then let `absURL` (or `relURL` if you like) the prepending of `www.example.com`.